backend/src/main/java/de/neuefische/cgnjava222/productgallery/service/FileService.java

```diff
@@ -40,7 +41,20 @@ public ImageInfo uploadPicture(MultipartFile file) {
         }
     }
 
-    public void deletePicture(List<String> id) throws Exception {
-        cloudinary.api().deleteResources(id, ObjectUtils.emptyMap());
+    public void deletePicture(List<String> ids) {
+        ApiResponse abc = tryDeleteResource(ids);
+        Map<String, String> object = (Map) abc.get("deleted");
+        String deleted = object.get("" + ids.get(0));
```

Comment:
```suggestion
        String deleted = object.get(ids.get(0));
```
ids is a list of strings, conversion by concatenation is not needed here